source/addComment.js

```diff
@@ -1,3 +1,27 @@
 $(document).ready(function(){
   // add your code here
+  function formChecker(comment, author) {
```

Comment:
since this function takes arguments, it'd be easy to extract it outside of the document ready function